File: mobile/app/src/main/java/com/sobczal2/biteright/data/api/responses/currencies/SearchResponse.kt
package com.sobczal2.biteright.data.api.responses.currencies

import com.sobczal2.biteright.dto.common.PaginatedList
import com.sobczal2.biteright.dto.currencies.CurrencyDto

data class SearchResponse(
    val currencies: PaginatedList<CurrencyDto>
)

File: backend/src/BiteRight.Application/Queries/Currencies/Search/SearchResponse.cs
// # ==============================================================================
// # Solution: BiteRight
// # File: SearchResponse.cs
// # Author: Łukasz Sobczak
// # Created: 20-02-2024
// # ==============================================================================

#region

using BiteRight.Application.Dtos.Common;
using BiteRight.Application.Dtos.Currencies;

#endregion

namespace BiteRight.Application.Queries.Currencies.Search;

public record SearchResponse(PaginatedList<CurrencyDto> Currencies);